src/snowflake/snowpark/functions.py

```diff
@@ -440,6 +440,17 @@ def current_region() -> Column:
     return builtin("current_region")()
 
 
+def current_account() -> Column:
+    """Returns the name of the the account where the current user is logged in.
```

Comment:
NIT:
```suggestion
    """Returns the name of the account used in the current session.
```

Snowflake defined what a Snowflake session is in docs: https://docs.snowflake.com/en/user-guide/session-policies.